demo/styles.css

```diff
@@ -152,4 +152,12 @@ nav a.router-link-active {
 }
 .mb-md {
   margin-bottom: 15px;
-}
\ No newline at end of file
+}
+
+.cell_red {
```

Comment:
lets keep kebab-case style convention for class name, so rename to .cell-red, .cell-green